backup-manager/symfony/2.1/config/packages/backup_manager.yaml

```diff
@@ -0,0 +1,9 @@
+# See full config at https://github.com/backup-manager/symfony
+bm_backup_manager:
```

Comment:
Just wondering whether the file must not (should not) be named `bm_backup_manager.yaml` then.

Reply:
Thank you